app/controllers/divisions_controller.rb

```diff
@@ -31,8 +31,14 @@ def index
         @year = params[:date]
       elsif params[:date] =~/^\d{4}-\d{2}$/
         @month = params[:date]
-      else
-        @date = params[:date]
+      elsif params[:date]
```

Comment:
> But I am not familiar with the code and I like baby steps. 

Totes :) I think that's what made me think this is a first step pointing towards a bigger win :)

So, looking at the test example you gave above, I think what's happening is that it's trying to get the Divisions page, but the code that finds the divisions is making an assumption that there are in fact divisions in the database. It's raising an error because there's no divisions being created in the test setup.

You can create divisions for this test using our [Division factory](). We use factory_girl for this http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Using_factories . Something like `create(:division)` will make you a valid division. You can use a Times loop to make a bunch.